src/generators/project/index.js

```diff
@@ -6,13 +6,65 @@ const getAllPaths = require('get-all-paths');
 const makeDir = require('make-dir');
 const requireg = require('requireg');
 const Generator = require('yeoman-generator');
+const ftp = require('basic-ftp');
 
 // Ours
 const { OUTPUT_DIRECTORY_NAME } = require('../../constants');
 const { cmd, hvy, opt, sec } = require('../../utils/color');
 const { success } = require('../../utils/logging');
 const { installDependencies } = require('../../utils/npm');
 const { combine } = require('../../utils/structures');
+const { getCredentials } = require('../../config/deploy');
+
+const DEPLOY_DIRECTORY = '/www/res/sites/news-projects/';
```

Comment:
Deploy directory isn't necessarily a constant. It would be good if (a) this path were not defined in multiple places (it's also [here](https://github.com/abcnews/aunty/blob/main/src/config/deploy.js#L23)) and, (b) at deploy time, it checked based on the actual deploy where `contentftp` is the deploy target in `.aunty/deploy.json`.

Reply:
OK after talking with @colingourlay I'm going to get the deploy directory elsewhere and delete this.